<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelWriter</class>
 <widget class="QWidget" name="ModelWriter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1362</width>
    <height>682</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,0">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Table:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit_filter"/>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_filter_clear">
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_all">
            <property name="text">
             <string>All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Merge Result:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QToolButton" name="toolButton_lineedit_all_clear">
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCodeEditor" name="textEdit_final"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>890</width>
        <height>662</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCodeEditor</class>
   <extends>QTextEdit</extends>
   <header location="global">QCodeEditor</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>lineEdit_filter</tabstop>
  <tabstop>toolButton_filter_clear</tabstop>
  <tabstop>toolButton_all</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>toolButton_lineedit_all_clear</tabstop>
  <tabstop>textEdit_final</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
